ui/js/models/repository.js

```diff
@@ -117,7 +117,9 @@ treeherder.factory('ThRepositoryModel', [
                     });
             };
 
-            _.each(repoNames, getStatus);
+            repoNames.forEach((repo) => {
```

Comment:
For cases like this, where the expression is short, I'd rewrite this to be on one line (the parenthesis and curly brackets can also then be removed). eg:

```js
repoNames.forEach(repo => getStatus(repo));
```